app/scripts/views/settings/two_step_authentication.js

```diff
@@ -128,6 +128,10 @@ const View = FormView.extend({
     return account.createTotpToken()
       .then(result => {
         this.$('.qr-image').attr('src', result.qrCodeUrl);
+
+        const qrImageAltText = t('Image of QR code used to setup two-step authentication or you can manually enter %(code)s into supported application.');
```

Comment:
The alt text doesn't need to describe the image as much [as provide an alternative to the image](https://en.wikipedia.org/wiki/Alt_attribute#Common_misconceptions). As such, I think it'd be alright to shorten the text and say "Use the code %(code)s to set up two-step authentication in supported applications."